### PR TITLE
docs: Add a debugging section to the IRQ load balancing doc.

### PR DESCRIPTION
**Pull request by marcel-apf**
Signed-off-by: Marcel Apfelbaum <marcel@redhat.com>

### CODE REVIEW

File: docs/irq-load-balancing.md [hunk 1]
```diff
@@ -58,4 +59,137 @@ spec:
...
+
+Start a pod configured as in [Design Details](#Design Details):
+The pod below is guaranteed and requires 2 exclusive CPUs out of the 6 available CPUs in the node.
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dynamic-irq-pod
+  annotations:
+     irq-load-balancing.crio.io: "true"
```
Comment (cynepco3hahue) [inline docs/irq-load-balancing.md:93]: can you change annotations values to `disable` one

File: docs/irq-load-balancing.md [hunk 2]
```diff
@@ -58,4 +59,137 @@ spec:
...
+     cpu-quota.crio.io: "true"
+spec:
+  containers:
+  - name: dynamic-irq-pod
+    image: "quay.io/openshift-kni/cnf-tests:4.5"
```
Comment (cynepco3hahue) [inline docs/irq-load-balancing.md:98]: let's use 4.6 or 4.7

File: docs/irq-load-balancing.md [hunk 3]
```diff
@@ -58,4 +59,137 @@ spec:
...
+2. Ensure the pod has its ```runtimeClassName``` as the respective profile name, in this case dynamic-irq-profile.
+3. Ensure the node selector targets a cnf-worker.
+
+Ensure the pod is running correctly.
+```
+oc get pod -o wide
+NAME              READY   STATUS    RESTARTS   AGE     IP             NODE                  NOMINATED NODE   READINESS GATES
+dynamic-irq-pod   1/1     Running   0          5h33m   10.135.1.140   cnf-worker.demo.lab   <none>           <none>
+```
+1. Ensure status is ```running```.
```
Comment (cynepco3hahue) [inline docs/irq-load-balancing.md:122]: running -> Running

File: docs/irq-load-balancing.md [hunk 4]
```diff
@@ -58,4 +59,137 @@ spec:
...
+/proc/irq/15/smp_affinity_list: 0
+/proc/irq/24/smp_affinity_list: 1
+/proc/irq/25/smp_affinity_list: 1
+/proc/irq/26/smp_affinity_list: 1
+/proc/irq/27/smp_affinity_list: 5
+/proc/irq/28/smp_affinity_list: 1
+/proc/irq/29/smp_affinity_list: 0
+/proc/irq/30/smp_affinity_list: 0-5
+```
+Note: Some IRQ controllers do not support IRQ re-balancing and will always expose the online CPUs as the IRQ mask.
```
Comment (cynepco3hahue) [inline docs/irq-load-balancing.md:182]: `will always expose the online CPUs` -> will always expose all online CPUs

File: docs/irq-load-balancing.md (hunk 1 above)
Comment (marcel-apf) [reply docs/irq-load-balancing.md:93]: sure

File: docs/irq-load-balancing.md (hunk 2 above)
Comment (marcel-apf) [reply docs/irq-load-balancing.md:98]: ok

File: docs/irq-load-balancing.md (hunk 3 above)
Comment (marcel-apf) [reply docs/irq-load-balancing.md:122]: done

File: docs/irq-load-balancing.md (hunk 4 above)
Comment (marcel-apf) [reply docs/irq-load-balancing.md:182]: done

**Pull request merged**